packages/main/src/Input.js

```diff
@@ -332,6 +332,14 @@ const metadata = {
 			type: Object,
 		},
 
+		_wrapperAccInfo: {
+			type: Object,
```

Comment:
is the _wrapperAccInfo needed? We deleted it recently, since it is not used anywhere.

Reply:
Done